doc/classes/AnimationNodeBlendSpace2D.xml

```diff
@@ -105,39 +119,49 @@
 			<argument index="1" name="pos" type="Vector2">
 			</argument>
 			<description>
+				Updates the position of the point at index [code]point[/code] on the blend axis.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="auto_triangles" type="bool" setter="set_auto_triangles" getter="get_auto_triangles">
+			If true, the blend space is triangulated automatically. The mesh updates every time you add or remove points with [method add_blend_point] and [method remove_blend_point].
 		</member>
 		<member name="blend_mode" type="int" setter="set_blend_mode" getter="get_blend_mode" enum="AnimationNodeBlendSpace2D.BlendMode">
+			Controls the interpolation between animations. See [code]BLEND_MODE_*[/code] constants.
 		</member>
 		<member name="max_space" type="Vector2" setter="set_max_space" getter="get_max_space">
+			The blend space's X and Y axes' upper limit for the points' position. See [method add_blend_point].
 		</member>
 		<member name="min_space" type="Vector2" setter="set_min_space" getter="get_min_space">
+			The blend space's X and Y axes' lower limit for the points' position.
```

Comment:
Should also have " See [method add_blend_point]." for consistency with other classes.